proposals/20200909-userpsace-deployment.md

```diff
@@ -0,0 +1,163 @@
+# Falco userspace launch support
+
+## Summary
+
+This is a proposal on how to address deployment of Falco in a userspace
+ environment.
+
+There has been a rise in popularity of serverless architectures that are 
+managed and make kernel instrumentation impossible. This makes it quite hard to
+monitor the runtimes. This project is about defining a patching language/definition
+that allows Falco to be deployed alongside a container in userspace.
+
+## Motivation 
+
+We want to run Falco inside containers without access to kernel space.
+
+By creating a new project (Kilt) - that defines a patching/deployment procedure 
+it is possible to deploy Falco in different userspace environments (k8s, cloud 
+formation, etc)
+
+The project consists of a definition for recipes to include binaries and alter 
+containers, a library to process recipes and runtimes that execute the recipes 
+on supported platforms.
+
+## Goals
+
+- Offer a definition format for patching/deploying Falco along user containers
+- Offer a golang library that parses and executes recipes
+- Define a patching file for Falco
+- Offer automation when using k8s (Admission Controller Mutating Webhook)
```

Comment:
These two last bullets (especially the last one) seem more use-cases than goals to me.

WDYT?

Reply:
Maybe you could add a section dedicated to specific use case implementations

Reply:
Added a small section and separated the 2 runtimes from the list.